### ISSUE TITLE
Issue with Firebase Cloud Message Plugins

### ISSUE THREAD

**Issue by hypesol** [ratson/cordova-plugin-admob-free#452]
This plugin is conflcting with cordova-plugin-firebasex and cordova-plugin-fcm-with-dependecy-updated. Your admob plugin and firebase cloud message plugins are not working at the same time.

**Comment by f2rhero** [comment]
yes, same issue with cordova-plugin-firebase-analytics and cordova-plugin-firebasex, for me admob.prepare() force close application

**Comment by 123dma** [comment]
Update the versions of the sdk to the Last on gradle

**Comment by pirpy76** [comment]
> Update the versions of the sdk to the Last on gradle

how to update?
